src/dev/display/fb/lv_linux_fbdev.c

```diff
@@ -26,7 +26,11 @@
     #include <linux/fb.h>
 #endif /* LV_LINUX_FBDEV_BSD */
 
-#include <lvgl/lvgl.h>
+#if defined(LV_LVGL_H_INCLUDE_SIMPLE)
```

Comment:
We should use a relative include, similar to [this](https://github.com/lvgl/lvgl/blob/master/src/dev/x11/lv_x11.h#L17).

Reply:
The <lvgl/lvgl.h> inclusion is not required for drivers